calculator/script.js

```diff
@@ -0,0 +1,105 @@
+document.addEventListener("DOMContentLoaded", function () {
+  var strCalc = "";
+  clearCalc();
+  document.querySelector(".btn-c").onclick = function () {
+    clearCalc();
+  };
+  document.querySelector(".btn-ce").onclick = function () {
+    if (strCalc.length > 1) {
+      strCalc = strCalc.slice(0, strCalc.length - 1);
+      updateCalc();
+    } else {
+      clearCalc();
+    };
+  };
+  document.querySelector(".btn-percent").onclick = function () {
+    strCalc += "%";
```

Comment:
you can check button value attribute
https://www.w3schools.com/tags/att_button_value.asp